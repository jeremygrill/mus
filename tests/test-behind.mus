Int main()
{
	Note start;
	Note be;
	Note hind;
	Chord when;
	Chord i;
	Chord leave;
	Chord the;
	Chord world;
	Chord worldd;
	Chord worlddd;
	Chord bee;
	Chord hindd;
	Seq s;

	start = (0, 0|3);
	be = (58, 50|1);
	hind = (63, 70|4);
	when = [(67, 50|1),(63, 30|1),(61, 50|1),(58, 50|1)];
	i = [(68, 50|1),(63, 30|1),(61, 50|1),(56, 50|1)];
	leave = [(70, 50|1),(63, 30|1),(61, 50|1),(55, 50|1)];
	the = [(67, 50|1),(63, 30|1),(61, 50|1),(58, 50|1)];
	world = [(65, 50|1),(63, 30|1),(60, 50|1),(56, 50|1)];
	worldd = [(67, 50|1),(63, 30|1),(60, 50|1),(55, 50|1)];
	worlddd = [(68, 50|1),(63, 30|1),(60, 50|1),(53, 50|1)];
	bee = [(65, 40|1),(63, 30|1),(59, 40|1),(56, 40|1)];
	hindd = [(67, 65|4),(63, 65|4),(58, 65|4),(51, 65|4)];


	s = [start] + [be];
	s = s + [hind];
	s = s + when;
	s = s + i;
	s = s + leave;
	s = s + the;
	s = s + world;
	s = s + worldd;
	s = s + worlddd;
	s = s + bee;
	s = s + hindd;

	prints(s);
	plays(s);
	return 0;
}