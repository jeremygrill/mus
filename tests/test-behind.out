$([(0,0|3)][(58,50|1)][(63,70|4)][(58,50|1)(61,50|1)(63,30|1)(67,50|1)][(56,50|1)(61,50|1)(63,30|1)(68,50|1)][(55,50|1)(61,50|1)(63,30|1)(70,50|1)][(58,50|1)(61,50|1)(63,30|1)(67,50|1)][(56,50|1)(60,50|1)(63,30|1)(65,50|1)][(55,50|1)(60,50|1)(63,30|1)(67,50|1)][(53,50|1)(60,50|1)(63,30|1)(68,50|1)][(56,40|1)(59,40|1)(63,30|1)(65,40|1)][(51,65|4)(58,65|4)(63,65|4)(67,65|4)])
